modules/View.js

```diff
@@ -372,63 +493,39 @@
     },
 
     /**
-     * Registers the child view if not already done so, then calls view.attach with the element argument
-     * @param $el {jQuery element} the element to attach to.
-     * @param view {View} the child view
-     * @param [options] {Object} optionals options object
-     * @param   [options.noActivate=false] {Boolean} if set to true, the child view will not be activated upon attaching.
-     * @method attachChildView
-     * @deprecated 0.3.x - use this.attachView($el, view, { shared: false }); instead
-     */
-    attachChildView: function($el, view, options) {
-      _.extend(options, { shared: false });
-      this.attachView($el, view, options);
-    },
-
-    /**
-     * @return {Boolean} true if this view has shared views
+     * @return {Boolean} true if this view has tracked views (limited by the options parameter)
      * @param [options={}] {Object}  Optional options.
-     *   @param [options.shared=false] {Boolean} The view is a shared view instead of a child view
-     *                                           (shared views are not disposed when the parent is disposed)
+     *   @param [options.shared=false] {Boolean} If true, only check the shared views. These are views not owned by this parent. As compared to a child view
+     *                                           which are disposed when the parent is disposed.
+     *   @param [options.child=false] {Boolean} If true, only check the child views. These are views that are owned by the parent and dispose of them if the parent is disposed.
      * @method hasTrackedViews
      */
     hasTrackedViews: function(options) {
-      var trackedViewsHash = this.__getTrackedViewsHash(options);
+      var trackedViewsHash = this.getTrackedViews(options);
       return !_.isEmpty(trackedViewsHash);
     },
 
     /**
-     * @return {Boolean} true if this view has child views
-     * @method hasChildViews
-     * @deprecated 0.3.x - use this.hasTrackedViews({ shared: false }); instead
-     */
-    hasChildViews: function() {
-      return this.hasTrackedViews({ shared: false });
-    },
-
-    /**
-     * @return all of the shared views this list view has registered
-     * @param [options={}] {Object}  Optional options.
-     *   @param [options.shared=false] {Boolean} The view is a shared view instead of a child view
-     *                                           (shared views are not disposed when the parent is disposed)
+     * Returns all tracked views, both child views and shared views.
      * @method getTrackedViews
+     * @param [options={}] {Object}  Optional options.
```

Comment:
If both are true, should we return everything? Intuitively I would expect that to happen, and it would also leave us in a better place in cases a third type is ever added. I know the docs say "only" and don't define that behavior, but arbitrarily returning one of them is unexpected.